day_6/exercises/burrito.rb

```diff
@@ -5,15 +5,43 @@
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
+
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping
+    p "Add sour cream."
+  end
+
+  def remove_topping
+    p "Remove guacamole."
+  end
+
+  def change_protein(protein)
+    self.protein = protein
+  end
 end
 
+
 dinner = Burrito.new("Beans", "Rice", ["cheese", "salsa", "guacamole"])
 p dinner.protein
 p dinner.base
 p dinner.toppings
+dinner.add_topping
+dinner.remove_topping
+dinner.change_protein("chicken")
+dinner.protein = "chicken"
+p dinner.protein
+
+# I'm finding this particular exercise very confusing. I'm not sure whether I'm
```

Comment:
Hey Carleigh! Thanks for the feedback and taking some time to talk out your thoughts here! So for this exercise we want you to add or remove whichever topping you pass into the `add_topping` or `remove_topping` methods. I see that you didn't pass anything in to either of them on this implementation, but the way you did it with `change_protein` is exactly what we want to see for that method. Ruby has some build in array methods that can handle the adding and removing of the topping within the method. Have you seen those before? If no this is a great chance to sharpen those googling skills! DM me if you have more questions about this.